Agents/ForecastingAgent/tests/conftest.py

```diff
@@ -143,11 +147,29 @@
 TEST_CASE_10 = 'TFT_OM_Quantity_with_Measure_with_Unit__no_overwriting'
 TEST_CASE_11 = 'TFT_OM_Quantity_without_Measure_with_Unit__overwriting'
 TEST_CASE_12 = 'TFT_OM_Quantity_without_Measure_with_Unit__no_overwriting'
+# Prophet with covariates test cases
+TEST_CASE_13 = 'Prophet_Covariates_OM_Quantity_without_scaling_with_Measure_without_Unit__overwriting'
```

Comment:
I believe that's wrong: `HeatDemand_1` is an `om:Quantity` and instantiated with a `om:Unit` in the test triples. Please fix those case descriptions to avoid confusion. Relevant cases are 1) Quantity with Measure and Unit and 2) Quantity without Measure without Unit